resources/tentacle.rb

```diff
@@ -19,28 +19,211 @@
 # limitations under the License.
 #
 
-actions :install, :configure, :register, :remove, :uninstall
+include OctopusDeploy::Tentacle
+
+resource_name 'octopus_deploy_tentacle'
+
+property :instance, kind_of: String, default: 'Tentacle'
```

Comment:
`kind_of`s are not needed any more and not mentioned in the docs for new resources